core-bundle/src/Resources/contao/languages/en/tl_image_size.xlf

```diff
@@ -56,6 +56,21 @@
       <trans-unit id="tl_image_size.skipIfDimensionsMatch.1">
         <source>Skip image processing if resizing is not needed. Note that no optimizations such as meta data stripping or JPEG compression will be applied in this case.</source>
       </trans-unit>
+      <trans-unit id="tl_image_size.formats.0">
+        <source>Convert image formats</source>
+      </trans-unit>
+      <trans-unit id="tl_image_size.formats.1">
+        <source>Select which format conversion should be applied.</source>
+      </trans-unit>
+      <trans-unit id="tl_image_size.formatsMissingWebp">
+        <source>WEBP images are disabled. You can enable them by adding "webp" to contao.image.valid_extensions.</source>
+      </trans-unit>
+      <trans-unit id="tl_image_size.formatsOptions.{&quot;webp&quot;:[&quot;webp&quot;,&quot;jpg&quot;]}">
```

Comment:
The encoding should not be necessary if you disabled entity encoding in the DCA ;)

Reply:
The encoding is because of the `xlf` file

Reply:
Ah👍🏻